zebra/zebra_vxlan.c

```diff
@@ -2183,6 +2188,12 @@ static int zebra_vxlan_handle_vni_transition(struct zebra_vrf *zvrf, vni_t vni,
 			zebra_evpn_send_add_to_client(zevpn);
 			zebra_evpn_read_mac_neigh(zevpn, ifp);
 		}
+
+		if (vlan_if) {
```

Comment:
Why not do this at line 2189 where vlan_if is found?

Reply:
Thanks, done.